src/teradata-connection.ts

```diff
@@ -1,23 +1,10 @@
 // Get the Teradata Native Binding
-let fastcall: any;
 /* tslint:disable-next-line */
-// let platform: string = (typeof window !== 'undefined' && (<any>window)['process']) ? 'electron' : 'node';
-let platform: string = 'node';
-let abiVersion: string = process.versions.modules;
-
-if (process.platform === 'win32') {
-  /* tslint:disable-next-line */
-  fastcall = require('@teradataprebuilt/fastcall-win32-' + platform + '-v' + abiVersion);
-} else if (process.platform === 'darwin') {
-  /* tslint:disable-next-line */
-  fastcall = require('@teradataprebuilt/fastcall-darwin-' + platform + '-v' + abiVersion);
-} else if (process.platform === 'freebsd') {
-  /* tslint:disable-next-line */
-  fastcall = require('@teradataprebuilt/fastcall-freebsd-' + platform + '-v' + abiVersion);
-} else if (process.platform === 'linux') {
-  /* tslint:disable-next-line */
-  fastcall = require('@teradataprebuilt/fastcall-linux-' + platform + '-v' + abiVersion);
-}
+let ffi: any = require('ffi-napi');
+/* tslint:disable-next-line */
+let ref: any = require('ref-napi');
+/* tslint:disable-next-line */
+let ArrayType: any = require('ref-array-di')(ref);
```

Comment:
Can we move the ArrayTpye initialization to where it was (i.e., inside TeradataConnection's constructor)?

Reply:
The initialization still happens in the constructor.  See line 46:
`this.byteArray = new ArrayType(this.ref.types.byte);`

Reply:
I was referring to "const ArrayType: any = fastcall.ArrayType;" was within the constructor. Wondering is there a reason that is counterpart "let ArrayType: any = require('ref-array-di')(ref);" is moved outside the constructor.